tutorials/quickstart/Solution/Billing/Billing.csproj

```
@@ -1,13 +1,23 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
-    <TargetFramework>net48</TargetFramework>
     <OutputType>Exe</OutputType>
-    <LangVersion>7.3</LangVersion>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+	<LangVersion>7.3</LangVersion>  
   </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Hosting" Version="5.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="5.0.0" />
+    <PackageReference Include="NServiceBus" Version="7.4.6" />
+    <PackageReference Include="NServiceBus.Extensions.Hosting" Version="1.1.0" />
+    <PackageReference Include="NServiceBus.Heartbeat" Version="3.0.1" />
+    <PackageReference Include="NServiceBus.Metrics.ServiceControl" Version="3.0.6" />
+    <PackageReference Include="NServiceBus.MicrosoftLogging" Version="1.3.1" />
```

Don't think we need NServiceBus.MicrosoftLogging anymore. See https://particular.net/blog/enhancements-in-nservicebus-hosting#Logging-integration

But also, to keep it simple, I think we can reduce it to only top-level packages and let the rest come in as transitive dependencies, even though we won't get latest versions. Then it's less to tell the user to do in Shipping.

I think this should work, will you double-check?
```suggestion
    <PackageReference Include="NServiceBus" Version="7.4.6" />
    <PackageReference Include="NServiceBus.Extensions.Hosting" Version="1.1.0" />
    <PackageReference Include="NServiceBus.Heartbeat" Version="3.0.1" />
    <PackageReference Include="NServiceBus.Metrics.ServiceControl" Version="3.0.6" />
```

---

Removing them didn't cause any issues.   Also, our Core 3.1 reference page on the website https://docs.particular.net/samples/netcore-reference/ mentions using the `NserviceBus.MicrosoftLogging` lib.